Series order: Model column Sort option to be taken in count

andr3i4o opened the pull request:
The browser is not order the columns defined in the model in the way they should be. Instead the columns are taken randomly and this probably is not desirable behavior.

ariacom commented:
Thank you for your request, it seems relevant and I will integrate it.
Can you consider also the changes for Plotly and ChartJS.
The sort will not have effect for series created by splitter values (e.g. 1 serie for each product categroy), so I would enhance this fix by adding .ThenByDescending or Ascending (i => i.SerieDisplayName).
Can you check this ?

andr3i4o commented:
Hello, thank you for the fast responce and the suggestions. I've updated the other files and added the additional sorting you've proposed. Please take a look.
